shell/platform/darwin/ios/framework/Headers/FlutterViewController.h

```diff
@@ -130,6 +130,13 @@ FLUTTER_EXPORT
  */
 - (id<FlutterPluginRegistry>)pluginRegistry;
 
+/**
+ * True if at least one frame has rendered and the ViewController has appeared.
+ *
+ * This property is reset to false when the ViewController disappears.
+ */
```

Comment:
I think this now does the same thing as what io.flutter.embedding.android.hasRenderedFirstFrame() does. Though it's named differently :D

Can we keep the names consistent?

Reply:
I'm not sure that name works here.  This can change from true to false depending on State and back to true again.

Reply:
Android does that too on the same instance. Though you could made an argument to change the Android name too. 

cc @matthew-carroll

Reply:
+1 for consistency. Or `isRenderingFirstFrame`, which can be used on both iOS and Android?

Reply:
I don't think `isRenderingFirstFrame` is quite right either - this currently works as indicating the state as to whether Flutter has rendered at least one frame since the view has become visible, or if the view is not visible and thus no frames will be rendered.

Reply:
`renderingFrames` seems problematic for multiple reasons. First, it is easily confused with a noun, e.g., these are some frames that are being rendered. Second, it doesn't have any sense of temporal context in the name. I think that `hasRenderedFirstFrame()` implies a somewhat specific place in time. There was a time before any frames were rendered, and then a time after the first frame was rendered, and this method tells if that moment has passed, as opposed to the somewhat ambiguous on-going "time of rendering frames". I think it's totally fine that `hasRenderedFirstFrame()` can toggle between true/false, so long as the reason is documented. That toggle is not arbitrary, it only occurs when specific resets or detachments occur. In other words a singular Flutter experience will never flip that value from true to false - only introducing a new Flutter experience, or detaching and re-attaching the original Flutter experience will result in that flag flipping.

Reply:
This part: "In other words a singular Flutter experience will never flip that value from true to false - only introducing a new Flutter experience, or detaching and re-attaching the original Flutter experience " really confused me.  Let's discuss offline

Reply:
The scenario most on my mind for this is showing the camera.

FlutterViewController appears -> isRenderingFrames = NO
Frame callback fires -> isRenderingFrames = YES
Camera starts -> isRenderingFrames = NO
Camera goes away -> isRenderingFrames = YES

In this situation, I find it confusing to think about which frame is the "first" frame, and I don't want to know if a frame "has" rendered - I want to know if at least one frame has rendered and Flutter could still potentially continue to render.

Reply:
Updated property name based on discussion - PTAL